tasks.py

```diff
@@ -0,0 +1,48 @@
+import json
+from typing import Dict
+import datetime
+
+
+def load_records(filepath):
+    with open(filepath) as user_records:
+        loaded_records = json.load(user_records)
+
+    return loaded_records
+
+
+class FullNameCreator():
```

Comment:
Just a tiny thing, but there's no need for parenthesis in python unless the class is inheriting from another class. So you can just have:
`class FullNameCreator:`